website/content/docs/ecs/manual/acl-controller.mdx

```diff
@@ -97,21 +75,21 @@ You must include the following top-level fields.
 
 In the `containerDefinitions` list, include one container with the following fields.
 
-| Field name    | Type    | Description                                                                                                                      |
-| -----------   | ------- | -------------------------------------------------------------------------------------------------------------------------------- |
-| `name`        | string  | The container name, which should be `acl-controller`                                                                             |
-| `image`       | string  | The `consul-ecs` image. Use our public AWS registry, `public.ecr.aws/hashicorp/consul-ecs`, to avoid rate limits.                |
-| `command`     | list    | Must be set as shown. The startup command for the ACL controller.                                                                |
-| `essential`   | boolean | Must be `true` to ensure the health of your application container affects the health status of the task.                         |
-| `secrets`     | list    | Must have `CONSUL_HTTP_TOKEN` set to the ACL bootstrap token and `CONSUL_CACERT_PEM` set to the Consul server CA certificate.    |
-| `environment` | string  | Must set the `CONSUL_HTTP_ADDR` environment variable to the address of the HTTP API of your Consul servers.                      |
+| Field name    | Type    | Description                                                                                                                                 |
+| -----------   | ------- | --------------------------------------------------------------------------------------------------------------------------------            |
+| `name`        | string  | The container name, which should be `acl-controller`                                                                                        |
+| `image`       | string  | The `consul-ecs` image. Use our public AWS registry, `public.ecr.aws/hashicorp/consul-ecs`, to avoid rate limits.                           |
+| `command`     | list    | Should be set as shown. The startup command for the ACL controller.                                                                         |
+| `essential`   | boolean | Must be `true` to ensure the health of your application container affects the health status of the task.                                    |
+| `secrets`     | list    | Must set `CONSUL_HTTP_TOKEN` to a Consul ACL token and, if applicable, set `CONSUL_CACERT_PEM` if a Consul server CA certificate is needed. |
```

Comment:
```suggestion
| `secrets`     | list    | Must set `CONSUL_HTTP_TOKEN` to a Consul ACL token and, if applicable, set `CONSUL_CACERT_PEM` if a Consul server CA certificate is needed. |
```
This description is still confusing. Are the values the two environment variables? Is this saying that you have to set the env vars first, then you can use them as values for this field?